kratos/linear_solvers/linear_solver.h

```diff
@@ -158,6 +158,7 @@ class LinearSolver
     */
     virtual void PerformSolutionStep(SparseMatrixType& rA, VectorType& rX, VectorType& rB)
     {
+        KRATOS_ERROR << "PerformSolutionStep is not implemented. Try using the Solve function." << std::endl;
```

Comment:
For consistency with existing code, I would suggest making the error message like https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/solving_strategies/schemes/residual_based_bdf_scheme.h#L230. I.e., "Calling base class method ...". Also in case it is not always appropriate to call Solve(), I would suggest omitting the hint "Try using ...".